tests/test_data/dobruchut.aktuality.sk/dobruchutaktualitysk.json

```diff
@@ -0,0 +1,46 @@
+{
+  "host": "dobruchut.aktuality.sk",
+  "canonical_url": "https://dobruchut.aktuality.sk/recept/79940/tvarohove-knedlicky-s-okamzitou-omackou/",
+  "site_name": "DobrúChuť.sk",
+  "author": "Howardcooks",
+  "language": "sk",
+  "title": "Tvarohové knedličky s okamžitou omáčkou",
+  "prep_time": 15,
+  "ingredients": [
+    "250 g hrudkovitý tvaroh",
+    "1 ks vajce",
+    "100 g polohrubá múka",
+    "1 štipka soľ",
+    "200 ml smotana na šľahanie",
+    "2 PL bazalkové pesto",
+    "50 g parmezán"
+  ],
+  "ingredient_groups": [
+    {
+      "ingredients": [
+        "250 g hrudkovitý tvaroh",
+        "1 ks vajce",
+        "100 g polohrubá múka",
+        "1 štipka soľ",
+        "200 ml smotana na šľahanie",
+        "2 PL bazalkové pesto",
+        "50 g parmezán"
+      ]
+    }
+  ],
```

Comment:
If `ingredient_groups` is difficult / not possible to support, we can remove this from the test expectation (no groupings are expected).

It would be nice to include support if possible - I notice that [this halloumi recipe](https://dobruchut.aktuality.sk/recept/79944/vyprazany-halloumi-syr-so-sladko-kyslou-omackou/) seems to have ingredient groupings - but I haven't checked the HTML yet to figure out how easy/difficult those might be to extract.

Reply:
Will take a closer look.

Reply:
Added.